### PR TITLE
Add log to debug #678

### PR DESCRIPTION
**Pull request by Yukikaze-CZR**
Add log to debug #678

### CODE REVIEW

**Review by youny626 — APPROVED** [review]

**Review by XuPeng-SH — APPROVED** [review]

File: core/src/index/knowhere/knowhere/index/vector_index/FaissBaseIndex.cpp [hunk 1]
```diff
@@ -70,6 +70,7 @@ FaissBaseIndex::SealImpl() {
     faiss::Index* index = index_.get();
     auto idx = dynamic_cast<faiss::IndexIVF*>(index);
     if (idx != nullptr) {
+        KNOWHERE_LOG_DEBUG << __FUNCTION__ << " IVF READONLY " << std::boolalpha << idx->is_readonly();
```
Comment (JinHai-CN) [inline core/src/index/knowhere/knowhere/index/vector_index/FaissBaseIndex.cpp]: Stop using __FUNCTION__, use another tag to replace it.

File: core/src/index/thirdparty/faiss/gpu/impl/IVFFlat.cu [hunk 2]
```diff
@@ -102,6 +106,16 @@ IVFFlat::copyCodeVectorsFromCpu(const float* vecs,
     if (stream != 0) {
         streamWait({stream}, {0});
     }
+    std::chrono::high_resolution_clock::time_point time5 = std::chrono::high_resolution_clock::now();
+
+    double span1 = (std::chrono::duration<double, std::micro>(time2 - time1)).count();
+    double span2 = (std::chrono::duration<double, std::micro>(time3 - time2)).count();
+    double span3 = (std::chrono::duration<double, std::micro>(time4 - time3)).count();
+    double span4 = (std::chrono::duration<double, std::micro>(time5 - time4)).count();
+    std::cout << "Span1: " << span1 * 0.001 << "ms" << std::endl;
```
Comment (JinHai-CN) [inline core/src/index/thirdparty/faiss/gpu/impl/IVFFlat.cu]: Shouldn't they already be microsecond?

File: core/src/index/thirdparty/faiss/gpu/impl/IVFFlat.cu [hunk 3]
```diff
@@ -102,6 +106,16 @@ IVFFlat::copyCodeVectorsFromCpu(const float* vecs,
...
+    std::cout << "Span2: " << span2 * 0.001 << "ms" << std::endl;
+    std::cout << "Span3: " << span3 * 0.001 << "ms" << std::endl;
+    std::cout << "Span4: " << span4 * 0.001 << "ms" << std::endl;
```
Comment (JinHai-CN) [inline core/src/index/thirdparty/faiss/gpu/impl/IVFFlat.cu]: Each search the time information will be printed out on stdout?

File: core/src/index/knowhere/knowhere/index/vector_index/FaissBaseIndex.cpp (hunk 1 above)
Comment (Yukikaze-CZR) [reply core/src/index/knowhere/knowhere/index/vector_index/FaissBaseIndex.cpp]: change __function__ to "Test before to_readonly:"

File: core/src/index/thirdparty/faiss/gpu/impl/IVFFlat.cu (hunk 2 above)
Comment (Yukikaze-CZR) [reply core/src/index/thirdparty/faiss/gpu/impl/IVFFlat.cu]: What we need is millisecond.

File: core/src/index/thirdparty/faiss/gpu/impl/IVFFlat.cu (hunk 3 above)
Comment (Yukikaze-CZR) [reply core/src/index/thirdparty/faiss/gpu/impl/IVFFlat.cu]: What we need is millisecond.

**Pull request merged**